artmanflow/steps/copy_script.py

```diff
@@ -0,0 +1,38 @@
+import subprocess
+
+staging_path = '/usr/local/google/home/vam/_/_/89/api-client-staging-private'
+dest_path = '/usr/local/google/home/vam/_/projects/github/vam-google/google-cloud-java'
+
+mapping = {
```

Comment:
This should not exist in artman. Artman's job is to take arbitrary specification and configuration and instrument what  needs to be run. It should not have "retail shop" information.

Reply:
Sorry, this copy script was committed accidentally, it is is not the actual part of artmanflow.